public/index.ts

```diff
@@ -24,7 +24,7 @@ import { OpenCellLoader } from "../src/loaders/OpenCellLoader";
 import { State, TestDataSpec } from "./types";
 import { getDefaultImageInfo } from "../src/Volume";
 
-const CACHE_MAX_SIZE = 500_000_000;
+const CACHE_MAX_SIZE = 1_000_000_000;
```

Comment:
MORE MORE MORE.  I just increased this so the whole time series could be cached. Didn't try to find a min value, just doubled it.

Reply:
Is doubling enough?

Reply:
enough to work to demonstrate the functionality with the sample data.  This is a tricky business because I am not aware of easy ways to reliably detect available mem resources in javascript.  We'll work out a lowest common denominator approach for production code.